code/modules/mob/living/bloodcrawl.dm

```diff
@@ -156,23 +158,22 @@
 	addtimer(CALLBACK(src, /atom/.proc/remove_atom_colour, TEMPORARY_COLOUR_PRIORITY, newcolor), 6 SECONDS)
 
 /mob/living/proc/phasein(obj/effect/decal/cleanable/B)
-	if(src.notransform)
+	if(notransform)
 		to_chat(src, "<span class='warning'>Finish eating first!</span>")
-		return 0
+		return FALSE
 	B.visible_message("<span class='warning'>[B] starts to bubble...</span>")
 	if(!do_after(src, 20, target = B))
 		return
 	if(!B)
 		return
 	forceMove(B.loc)
-	src.client.eye = src
-	src.visible_message("<span class='warning'><B>[src] rises out of the pool of blood!</B></span>")
+	client.eye = src
+	visible_message("<span class='boldwarning'><B>[src] rises out of the pool of blood!</B></span>")
```

Comment:
should get rid of the <B> </B> if you're using bold warning

Reply:
Right, removed.